libavcodec/vaapi_decode.c

```diff
@@ -666,7 +653,8 @@ int ff_vaapi_decode_init(AVCodecContext *avctx)
     ctx->hwctx  = ctx->device->hwctx;
 
     err = vaapi_decode_make_config(avctx, ctx->frames->device_ref,
-                                   &ctx->va_config, avctx->hw_frames_ctx);
+                                   &ctx->va_config, avctx->hw_frames_ctx,
```

Comment:
this changed the original behavious.  It will give 16 to mpeg2. but the previous code only gives 2.